Selenium/src/Life360/DownloadLife360Button.java

```diff
@@ -0,0 +1,45 @@
+package Life360;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+import java.util.ArrayList;
+
+/*
+Title:DownloadLife360Button
```

Comment:
Change the title to DownloadLife360Button -> Validate DownloadLife360 for free Text on Download screen